### PR TITLE
Rename EXT_feature_metadata → EXT_mesh_features

### PR DESCRIPTION
**Pull request by donmccurdy**
Replaces https://github.com/CesiumGS/glTF/pull/19, following discussion in #17.

Will likely want some additional rephrasing, beyond the find/replace here, but will do that in a separate PR.

### CODE REVIEW

File: extensions/2.0/Vendor/EXT_mesh_features/schema/gltf.EXT_feature_metadata.schema.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (ptrgags) [inline extensions/2.0/Vendor/EXT_mesh_features/schema/gltf.EXT_feature_metadata.schema.json:1]: Make sure the name of this file gets updated to `gltf.EXT_mesh_features.schema.json` -- there are a couple other files like this too.

**Review by ptrgags — COMMENTED** [review]
@donmccurdy looks good, though again, some files  include the extension name in the filename, make sure to rename those too.

I'm also going to hold off on merging until we discuss https://github.com/CesiumGS/3d-tiles/pull/483/files#r714202819 later today.

**Comment by donmccurdy** [comment]
> some files include the extension name in the filename...

Resolved. 👍

> I'm also going to hold off on merging until we discuss...

Sounds good!

**Comment by ptrgags** [comment]
Merging, thanks @donmccurdy!

**Pull request merged**